CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# v1.18.0
+:new: What's new:
+
+- LakeFSFS - TokenProvider AWS (no login yet) (#7604)
```

Comment:
What does no login yet mean? Is it the remote login described in the next item?

Reply:
Yes